### ISSUE TITLE
Bronze Boiler Infinite Fuel

### ISSUE THREAD

**Issue by shawnbyday** [TheLimePixel/GregBlock#196]
For some reason my bronze boiler has been running non-stop on one piece of charcoal for the last 3 real life days. Not sure if this is a bug here or upstream.

**Comment by programgames** [comment]
Yes it's a problem of the old gregtech CE version in the modpack, just update the modpack to the 2.2.1 version

**Comment by programgames** [comment]
I had the same case

**Comment by programgames** [comment]
What is the version of your modpack ?

**Comment by shawnbyday** [comment]
I think I'm still on 2.2.0. I'll check into that when I get home, thanks!
